app/src/main/res/layout/component_bookmark.xml

```diff
@@ -20,17 +20,38 @@
             app:layout_constraintTop_toTopOf="parent"
             tools:listitem="@layout/library_site_item" />
 
-    <TextView
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Can we remove the extra constraintLayout along with the constraints on the children and just add width/height as `wrap_content` on the sign-in button? The gravity is already set on the children

Reply:
Yes good call!